app/models/notification.rb

```diff
@@ -40,6 +40,19 @@ def send_new_follower_notification_without_delay(follow, is_read: false)
       Notifications::NewFollowerWorker.new.perform(follow_data, is_read)
     end
 
+    def send_to_mentioned_users_and_followers(notifiable, _action = nil)
+      return unless notifiable.is_a?(Article) && notifiable.published?
+
+      # Mentions::CreateAll will check to see if there are any @-mentions in the post, and will create the associated
```

Comment:
**suggestion (non-blocking):** I would probably add this as class-level documentation to `Mentions::CreateAll` instead which also has the benefit that it will get picked up by YARD.

```ruby
module Mentions
  # This class will check to see...
  class CreateAll
```

Reply:
Wow, I didn't realize YARD would pick these up! (Clearly, I didn't know about or use that feature).

I like the idea of making this class-level documentation 👍🏽